cooltools/cli/coverage.py

```diff
@@ -49,6 +49,14 @@
     is_flag=True,
     default=False,
 )
+@click.option(
+    "--clr_weight_name",
+    help="Name of the weight column. Specify if wanting to calculate"
+    " balanced coverage. In this case, cis counts will not be stored.",
+    type=str,
+    default=None,
+    show_default=False,
+)
 @click.option(
```

Comment:
we might want to expose that prefix thing as a CLI argument  - maybe not - not sure ... just to provide flexible naming 
@gfudenberg @Phlya ?

Reply:
mmm, yeah seems complicated... 

changing store to store_in_cooler would definitely clarify the use of that argument vs. the output argument though for the CLI!

Reply:
Yeah I think it could be nice to be able to specify in CLI. Let me know if others agree and I can add it.